### PR TITLE
Add limits, monthly_counts and reset time to get_keys method

### PR DESCRIPTION
**Pull request by ChaituVR**
Merging this should not break anything

How to test:
- Run `docker compose up`
- Run (make sure to change the `secret` header to same one as the secret in your .env)
```
curl --location 'http://localhost:3002/' \
--header 'secret: 1234' \
--data '{
    "jsonrpc": "2.0",
    "method": "get_keys",
    "params": {
        "app": "snapshot-hub"
    },
    "id": "123456"
}'
```

See the new fields in output

### CODE REVIEW

File: src/methods.ts [hunk 1]
```diff
@@ -73,8 +73,6 @@ export const logReq = async (key: string, app: string) => {
 
     if (!keyData?.key) return { error: 'Key does not exist', code: 401 };
     if (!keyData?.active) return { error: 'Key is not active', code: 401 };
-    if (keyData?.month_total >= limits[app].monthly)
-      return { error: 'Key is restricted for this month', code: 429 };
```
Comment (ChaituVR) [inline src/methods.ts:77]: Even if the count exceeds, we keep counting, but the API key will work with default rate limit

File: src/methods.ts [hunk 2]
```diff
@@ -88,12 +86,23 @@ export const getKeys = async (app: string) => {
   try {
     if (!apps.includes(app)) return { error: 'App is not allowed', code: 401 };
     const activeKeys = await getActiveKeys(app);
+    // Reset timestamp is the first day of the next month
+    const resetTimestamp = (
+      new Date(new Date().getFullYear(), new Date().getMonth() + 1, 1).getTime() / 1e3
```
Comment (wa0x6e) [inline src/methods.ts:91]:
```suggestion
      Date.UTC(new Date().getFullYear(), new Date().getMonth() + 1, 1) / 1e3
```

Comment (wa0x6e) [reply src/methods.ts:91]: Date manipulation does not take timezone into account

**Review by wa0x6e — APPROVED** [review]
tAck

**Pull request merged**